.ai/auto-review/github_reviewer.py

```diff
@@ -92,11 +93,31 @@ def main():
             for response in responses:
                 if response.line:
                     result = post_line_comment(github=github, file=file, text=response.text, line=response.line)
+                if not result:
+                    diff_hunks = extract_diff_hunks(file_diffs)
+                    result = post_line_comment_with_diff_hunks(github=github, file=file, text=response.text, line=response.line, diff_hunk=diff_hunks)
                 if not result:
                     result = post_general_comment(github=github, file=file, text=response.text)
```

Comment:
100 : Incorrect argument: `post_line_comment_with_diff_hunks` expects a string for `diff_hunk`, but `extract_diff_hunks` returns a list of strings. This will lead to a TypeError. : Pass the correct diff hunk string to the function or modify the function to accept list of diff hunks.